Failed to building sample of  oneCCL_Getting_Started for GPU and CPU

# Summary
Failed to build oneCCL_Getting_Started for gpu with provided steps.
while manual bulid works with icpx -fsycl src/sycl/sycl_allreduce_test.cpp -I./src/include -lccl -lmpi.

# Version
oneAPI2024

# Environment
Ubuntu 22.04

# Steps to reproduce
source ${ONEAPI_ROOT}/setvars.sh --ccl-configuration=cpu_gpu_dpcpp
 
cd oneapi-toolkit/oneCCL/oneCCL_Getting_Started
mkdir build
cd build
cmake ..  -DCMAKE_C_COMPILER=icx -DCMAKE_CXX_COMPILER=icpx -DCOMPUTE_BACKEND=icpx
make sycl_allreduce_test

# Observed behavior
$ make sycl_allreduce_test
make: *** No rule to make target 'sycl_allreduce_test'.  Stop.

# Expected behavior
Build PASS

--- a/Libraries/oneCCL/oneCCL_Getting_Started/README.md
+++ b/Libraries/oneCCL/oneCCL_Getting_Started/README.md
@@ -119,7 +119,7 @@ Users can rebuild the cpu_allreduce_test.cpp by typing "make cpu_allreduce_test"
   cd oneapi-toolkit/oneCCL/oneCCL_Getting_Started
   mkdir build
   cd build
-  cmake ..  -DCMAKE_C_COMPILER=icx -DCMAKE_CXX_COMPILER=icpx -DCOMPUTE_BACKEND=icpx
+  cmake -E env COMPUTE_BACKEND=dpcpp cmake .. -DCMAKE_C_COMPILER=icx -DCMAKE_CXX_COMPILER=icpx -DCOMPUTE_BACKEND=dpcpp
   make sycl_allreduce_test
   ```
 > NOTE: The source file "sycl_allreduce_test.cpp" will be copied from ${INTEL_ONEAPI_INSTALL_FOLDER}/ccl/latest/examples/sycl to build/src/sycl folder.

